.github/workflows/reusable_build.yml

```diff
@@ -92,13 +92,11 @@ jobs:
                   fi
               done
               for crate in "${crates[@]}"; do
-                  echo "==> Updating $crate"
-                  cargo +$RUST_NIGHTLY update $crate
                   echo "==> Checking $crate version"
                   latest_version=$(cargo +$RUST_NIGHTLY search $crate | grep "^$crate" | awk '{print $3}' | tr -d '"')
                   tree_version=$(cargo +$RUST_NIGHTLY tree --depth 1 --prefix none | grep $crate | awk '{print $2}' | tr -d 'v')
                   if [[ $latest_version != $tree_version ]]; then
-                      echo "Failure when updating $crate to $latest_version"
+                      echo "$crate version is $tree_version whereas $latest_version is available; please update !"
```

Comment:
To my mind, we should only enforce the versions of our SDK crates.

Reply:
Ok I missed this `for crate in "ledger_secure_sdk_sys" "ledger_device_sdk" "include_gif"; do`
Ok for me